citar-org-roam.el

```diff
@@ -174,20 +195,28 @@ space."
           (propertize title 'face 'citar))
          (gethash citekey cands))))))
 
+(defun citar-org-roam--make-info-plist (citekey)
+  "Return org-roam capture template plist for CITEKEY."
+  (let* ((infopl)
+         (entry (citar-get-entry citekey)))
+     (cl-loop for (key value)
+              on citar-org-roam-template-fields
+              by 'cddr
+         do (setq infopl
+             (plist-put infopl
+                        key
+                        (cdr (citar-get-field-with-value value entry)))))
+     (setq infopl (plist-put infopl :citekey citekey))
+     infopl))
+
 (defun citar-org-roam--create-capture-note (citekey entry)
   "Open or create org-roam node for CITEKEY and ENTRY."
   ;; adapted from https://jethrokuan.github.io/org-roam-guide/#orgc48eb0d
-  (let ((title (citar-get-value "title" entry))
-        (key citar-org-roam-capture-template-key)
-        (author (or (citar-get-value "author" entry)
-                    (citar-get-value "editor" entry)))
-        (type (citar-get-value "=type=" entry))
-        (pages (citar-get-value "pages" entry))
-        (year (or (citar-get-value "year" entry)
-                  (citar-get-value "date" entry)
-                  (citar-get-value "issued" entry))))
+  (let* ((title (citar-format--entry
+                 citar-org-roam-note-title-template entry))
+         (key citar-org-roam-capture-template-key))
     (apply 'org-roam-capture-
-           :info (list :citekey citekey :author author :type type :pages pages :year year)
+           :info (citar-org-roam--make-info-plist citekey) ; add the or title here too?
            :node (org-roam-node-create :title title)
```

Comment:
This in the end is the key question: should this be:

1. the `:citar-title` value (though given this is configurable, we can't count on that)?
2. the value created according to `citar-org-roam-title-template`?
3. some mechanism via `org-roam-capture` templates?

Status quo with this PR is 2.

If it should be 3, I don't know how to do that.

Reply:
I would go for the citar-title (e.g. the title inside the bib entry). Combining it with author, ... can then easily be done in the capture template itself.

Reply:
Then inside the templating we can use both the title, as configured using `citar-org-roam-note-title-template`, or the title as in the bib entry.

Reply:
To clarify, I'm talking about the title that shows up in the `org-roam` db, and in the UI; not in the note file. 

Does that change anything you wrote above?

Reply:
Like, look at how ORB does this; programmatically, using `bibtex-completion` (well, with a wrapper for it I guess):

https://github.com/org-roam/org-roam-bibtex/blob/c87acc2d5685ba2608e2bd1f128e723f1de9d837/org-roam-bibtex.el#L603-L610

Reply:
Yeah, the very fact we have this confusion between the two of us illustrate the problem.

OK, how about this:

1. change `citar-org-roam-title-template` to `citar-org-roam-node-title-template` (or can be your "db"; it's just it's also a UI thing
2. define `citar-node-title` (or "db") as the title used on L221; the default `citar-title` can then be used in templates for finer-grained formatting

I think that should solve it?